Tools/autotest/sim_vehicle.py

```diff
@@ -1028,6 +1041,10 @@ def generate_frame_help():
 group_sim.add_option("--disable-ekf3",
                      action='store_true',
                      help="disable EKF3 in build")
+group_sim.add_option("", "--start-time",
+                     default=None,
+                     type='string',
+                     help="specify simulation start time in format YYYY-MM-DD-HH:MM")
```

Comment:
This should mention the time is in UTC.

Perhaps in the future we could use the usual `+10` notation to specify a timezone.

Reply:
The time string is actually assumed to be in the local time zone - I've clarified this in the option help

I'll look into supporting specification of timezone, the library I've used so far doesn't support this on my system.

Reply:
I think it's pretty clear now... but I guess "local time" could mean either of the user's local time or the simulated Plane's local time :-)

Reply:
That's true! Now "your local time zone"